cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Actions.hs

```diff
@@ -340,8 +340,8 @@ dpGovActionProtocolParametersUpdate = \case
   ShelleyBasedEraConway ->
     ConwayEraBasedProtocolParametersUpdate
       <$> pCommonProtocolParameters
+      <*> pIntroducedInBabbagePParams   
```

Comment:
In the definition of `EraBasedProtocolParametersUpdate` data type, [babbage params are defined after alonzo params](https://github.com/IntersectMBO/cardano-api/blob/03de67a58e044c41c62ab8683bffac52771bcd37/cardano-api/internal/Cardano/Api/ProtocolParameters.hs#L239), so your change here is not correct.